phpini/list_pkgs.cgi

```diff
@@ -40,24 +39,26 @@ if (@pkgs) {
 			$pkg->{'ver'},
 			$pkg->{'phpver'},
 			$pkg->{'binary'},
-			$vmap ? ( $pkg->{'shortver'}, $users ) : ( ),
+			$vmap ? ( $users ) : ( ),
 			], \@tds, "d", $pkg->{'name'});
-		$got{$pkg->{'name'}}++;
+		$got{$pkg->{'phpver'}}++;
 		}
 	print &ui_columns_end();
 	print &ui_form_end([ [ undef, $text{'pkgs_delete'} ] ]);
 	}
 else {
 	print "<b>$text{'pkgs_none'}</b> <p>\n";
 	}
-
-if (&foreign_installed("package-updates")) {
+my @newpkgs = grep { !$got{$_->{'phpver'}} } &list_best_available_php_packages();
```

Comment:
list_best_available_php_packages calls list_available_php_packages which depends on the package-updates module, so it's not safe to call this before the foreign_installed check.

Reply:
Got it! Fixed now!